package.json

```diff
@@ -198,10 +198,10 @@
     ]
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": [
+    "*.{mjs}": [
       "npx eslint"
     ],
-    "*.scss": [
+    "*.sass": [
```

Comment:
[curious] Can you clarify these changes a bit? Why are they needed?

Reply:
I needed to ignore the pre-commit hook)
This is a temporary change that I will remove 💯